site/src/examples/date-picker/Default.tsx

```diff
@@ -0,0 +1,6 @@
+import { ReactElement } from "react";
+import { DatePicker } from "@salt-ds/lab";
+
+export const Default = (): ReactElement => (
+  <DatePicker style={{ width: "256px" }} />
```

Comment:
I wonder if we should try to avoid adding inline styles directly to the components in our examples, if we're doing it for others then I guess it's fine

Reply:
looks like it is standard in other examples so it doesn't take 100%w ...  i guess we could also make a wrapper but for now i took the same size as the input examples